tools/setup/generate_zulip_bots_static_files.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+import sys
+import shutil
+from os.path import basename, dirname, splitext
```

Comment:
Stylistically, for all stdlib modules, our convention is to simply import `os.path` and access these with the full name.

Reply:
I opened https://github.com/zulip/zulip/issues/6543 for making this a lint rule.